packages/venia-ui/lib/components/CheckoutPage/checkoutPage.css

```diff
@@ -77,11 +77,21 @@
 }
 
 .summaryContainer {
-    grid-column: 2 / span 1;
-    grid-row: 1 / span 1;
-    height: 0;
-    position: sticky;
-    top: 5.5rem;
+    grid-column: 1 / span 1;
+}
+
+@media (min-width: 961px) {
+    .summaryContainer {
+        grid-column: 2 / span 1;
+        grid-row: 2 / span 3;
+        position: sticky;
+        top: 5.5rem;
+        height: min-content;
+    }
+
+    .signin_container ~ .summaryContainer {
```

Comment:
We try to avoid descendent selectors like this. Can we target the container another way?

Reply:
We need `.summaryContainer` to be aligned with shipping information, but in case of guest checkout login section is added (+1 row), so we need to move summary down one row, and using such selector is actually solves this.

Reply:
True. We should probably just use a media query instead.